api/src/main/java/org/pmiops/workbench/interactiveanalysis/InteractiveAnalysisService.java

```diff
@@ -122,15 +121,11 @@ public String localize(
     String googleProjectId = dbWorkspace.getGoogleProject();
     // Use current dir if not Jupyter
     String editDir = isGceRuntime ? "workspaces/" + workspacePath : "";
-    // note: Playground Mode only applies to Jupyter
-    String playgroundDir = "workspaces_playground/" + workspacePath;
-    String targetDir = isPlayground ? playgroundDir : editDir;
 
     var storageLink =
         new StorageLink().cloudStorageDirectory(gcsNotebooksDir).localBaseDirectory(editDir);
 
     if (isGceRuntime) {
-      storageLink.localSafeModeBaseDirectory(playgroundDir);
```

Comment:
Is there any need for edit mode needing access to the safe mode directory?

Reply:
nope..they're separate